ethsigner/core/src/test/java/tech/pegasys/ethsigner/core/requesthandler/sendtransaction/transaction/EeaTransactionTest.java

```diff
@@ -22,21 +22,25 @@
 import tech.pegasys.ethsigner.core.jsonrpc.JsonRpcRequestId;
 
 import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
 import java.util.List;
 
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.web3j.crypto.Sign.SignatureData;
 import org.web3j.protocol.eea.crypto.PrivateTransactionDecoder;
 import org.web3j.protocol.eea.crypto.SignedRawPrivateTransaction;
+import org.web3j.rlp.RlpString;
 import org.web3j.utils.Numeric;
 
 public class EeaTransactionTest {
+
   private EeaTransaction eeaTransaction;
+  private EeaSendTransactionJsonParameters params;
 
   @BeforeEach
   public void setup() {
-    final EeaSendTransactionJsonParameters params =
+    params =
         new EeaSendTransactionJsonParameters(
             "0x7577919ae5df4941180eac211965f275cdce314d",
             "ZlapEsl9qDLPy/e88+/6yvCUEVIvH83y0N4A6wHuKXI=",
```

Comment:
Shouldn't the privateFrom and privateFor values be updated to use the base64 format?
Given that it's not a hex string or base64 I would have expected this to fail with an error.